scrapers/peeonher.yml

```diff
@@ -0,0 +1,28 @@
+name: peeonher
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - peeonher.com/updates/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //h1[@class='page_title']
+     Date:
+        selector: //strong[text()="Published"]/following-sibling::text()
+        postProcess:
+          - replace:
+            - regex: (\d+)[dhnrst]+
+              with: $1
+          - parseDate: 2 January 2006
+     Performers:
+      Name: //div[@class="pull-left"]//a/text()
+     Details: //div[@class='con_left']/p[1]
```

Comment:
/p`[1]`
Was there a scene that needed that? I tried some scenes but could only find single paragraphed ones.

Reply:
Hmmm yeah don't know why I had that in there it serves no purpose at all.

Reply:
I wasnt sure for the `[1]` part ( whether to use  `//div[@class='con_left']/p`  and concat also.)
The `/p` is needed though because otherwise we get the **Story ** part with every description
It should be `//div[@class='con_left']/p`